db/migrations/2022062313540000_add_image_framing_mode.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const { irreversibleMigration } = require('../utils');
+
+module.exports = {
+  up: async (queryInterface) => {
+    // collection_id = 36 := SARI
+    await queryInterface.sequelize.query(`
+        UPDATE images SET framing_mode = 'single_image' WHERE framing_mode IS NULL;
+        UPDATE images SET framing_mode = 'composite_image' WHERE collection_id = 36;
```

Comment:
can you remove that? Collection_id may be different on beta and production since we have some collections only on beta. Also code should be data agnostic for people not targeting smapshot.heig-vd.ch